packages/teams-components/src/components/Button/validateIconButton.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * @throws Error if icon button is missing required props
+ */
+export const validateIconButton = (props: {
+  icon?: unknown;
+  children?: unknown;
+  tooltip?: unknown;
+  'aria-label'?: string;
+  'aria-labelledby'?: string;
+}) => {
+  if (
+    !props.children &&
+    props.icon &&
+    !props.tooltip &&
```

Comment:
Should be this `Tooltip` or `title`?

Reply:
let's stick with title - I'm gonna go with the thought that our tooltip for buttons is basically an enhanced DOM title